deploy/main.tf

```diff
@@ -17,9 +17,13 @@ resource "random_string" "rand4" {
   upper   = false
 }
 
+######################################
+# Generate Tags
+######################################
+
 module "vpc_label" {
-  source    = "aws-ia/label/aws"
-  version   = "0.0.2"
+  source    = "aws-quickstart/label/aws"
```

Comment:
nothing should be referencing aws-quickstart (use aws-ia)

Reply:
Updated!